graphium/data/sampler.py

```diff
@@ -0,0 +1,43 @@
+import torch.utils.data as data_utils
+import numpy as np
+import time
+import os
+import torch
+
+
+class CustomSampler(data_utils.Sampler):
```

Comment:
I suggest changing the name from `CustomSampler` to `DatasetSubSampler` or something similar that tells you more about the class